hist/hist/src/TH1.cxx

```diff
@@ -106,7 +106,7 @@ ROOT supports the following histogram types:
       - TH1C : histograms with one byte per channel.   Maximum bin content = 127
       - TH1S : histograms with one short per channel.  Maximum bin content = 32767
       - TH1I : histograms with one int per channel.    Maximum bin content = INT_MAX (\ref intmax "*")
-      - TH1F : histograms with one float per channel.  Maximum precision 7 digits
+      - TH1F : histograms with one float per channel.  Maximum precision 7 digits, maximum bin content = +/-1e7
```

Comment:
What is the reason the number `1e7` was chosen? Naively, I would expect the maximum of a float bin to be `std::numeric_limits<float>::max()`.

Reply:
Nice, thanks a lot for the pointer! I would then suggest adding a comment similar to yours here https://github.com/root-project/root/issues/6671#issuecomment-1862319656 to convey the same information briefly also in the docs

Reply:
added!